DrawingEngine/DrawingEngine.h

```diff
@@ -0,0 +1,25 @@
+#ifndef DRAWINGENGINE_H
+#define DRAWINGENGINE_H
+
+#include <iostream>
+#include <memory>
+#include <any>
+#include <vector>
+#include "../Logs/Logs.h"
+#include "../ClassFactory/ClassFactory.h"
+
+using namespace std;
+
+class DrawingEngine : public ClassFactory {
+private:
+	 shared_ptr<int> width = make_shared<int>();
```

Comment:
why not just ints, or even better unsigned ints as the width and height shall not be negative